tags/tags.toml

```
@@ -803,3 +803,18 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[master-events]
+keywords = ["master-events", "master events", "me", "v13-events", "v13 events"]
```

```suggestion
keywords = ["master-events", "master events", "v13-events", "v13 events"]
```

Def. remove the `me` one here, too ambigious